src/components/assets/GraphCMSAsset.test.tsx

```diff
@@ -35,7 +35,7 @@ describe('GraphCMSAsset', () => {
         const id = 'test';
         const title = 'Test asset';
         const classes = 'test-class1 test-class2';
-        const style = { paddingTop: 5 };
+        const style = { paddingTop: '5' };
```

Comment:
A number should be a valid input, I'd prefer not to change those kinds of things if it's avoidable (reduce work between presets)

Reply:
I think it can be avoided, see https://github.com/frenic/csstype#generics

Reply:
Yeah it can be avoided, I just wanted to use the standard pattern, I'll change it.